src/lorarmesh.cpp

```diff
@@ -1,5 +1,22 @@
+#include <Layer1_LoRa.h>
+#include <LoRaLayer2.h>
 #include "loramesh.h"
 
+#define ADDR_LENGTH 4
+#define LORA_CS 18
+#define LORA_RST 14
+#define LORA_IRQ 26
+#define LORA_FREQ 915E6
+#define LED 25
+#define TX_POWER 20
+
+#define DATAGRAM_HEADER 5
+#define MESSAGE_LENGTH 233
+#define CONNECT_DELAY_MILLS 500
+#define MAX_CONNECT_RETRIES 10
```

Comment:
**Connect** delay and **connect** retries? Layer1->init() doesn't seem to connect anywhere. Please rename it.

Reply:
They were replaced by:
`BOOT_LL1_DELAY_MICROS 500`
`MAX_BOOT_LL1_RETRIES 10`